assessment/libs/finance2.php

```diff
@@ -533,6 +533,7 @@ function fin_DBD($D1,$D2) {
 // An embeded iframe code for inserting the solver into a question.
 
 function fin_TVM() {
+	return '<iframe src="https://p2.amirtavangar.com" name="TVMsolver" width="420" height="680" frameborder="0" scrolling="no" ><p>Your browser does not support iframes.</p></iframe>';
```

Comment:
Please remove this change from the pull request, since it's not related to the complex number stuff.